checkpoint2/index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Ryan's Checkpoint 2</title>
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+      <div class="background-image">
+    <header>
+      <h1>Ryan Roberts</h1>
+    </header>
+    <nav>
+        <a href="#about">About</a> &emsp;
+        <a href="#jobs">Professional Experience</a> &emsp;
+        <a href="#skills">Skills</a> &emsp;
+        <a href="#school">Education</a> &emsp;
+        <a href="#contact">Contact</a>
+    </nav>
+    <main>
+      <div class="photo">
+        <img src="ryan.jpg">
+      </div>
+      <div class="flexbox-container1">
+      <div class="flexbox-container2">
+        <div class="about" id="about">
+          <section>
+            <h2>About</h2>
+              <article>Highly knowledgeable and results-oriented  professional, offering wide-ranging experience across the technology and sales and marketing industries. Proven ability to increase revenue and acquire new accounts as well as in closing business negotiations. Exhibits superior talents in creating effective marketing strategies and plans to increase sales, capitalize on growth opportunities, and maintain profitable business operations. Adept at establishing win-win relationships and addressing diverse customer needs by providing innovative solutions and ensuring utmost satisfaction. Equipped with persuasive verbal and written communication skills.</article>
+          </section>
+        </div>
+        <div class="skills" id="skills">
+          <section>
+            <h2>Skills</h2>
+              Sales <br>   Product Marketing  <br>  SaaS <br>
+              Customer Service <br>  Client Development		<br>	Training/Teaching  <br>  New Business Development	<br>	Team Leadership	<br>  Microsoft Office <br>  Customer Retention		<br>Marketing Strategy		<br>	Hardware/Software  <br> Business to Business  <br> Account Management		<br>	Leadership Development <br> HTML	<br>	CSS
+          </section>
+        </div>
+        <div class="school" id="school">
+          <section>
+            <h2>Education</h2>
+              <h3>Austin Coding Academy</h3>
+                101-Intro to the Web Field Of Study Web Page, Digital/Multimedia and Information Resources Designs 2017
+              <h3>Libery University</h3>
+                Master of Arts in Theological Studies 2013
+              <h3>California State University-Fresno</h3>
+                Bachelor of Arts in History 1999
+          </section>
+        </div>
+        <div class="contact" id="contact">
+          <section><h2>Contact</h2>
+            Mobile: (559) 917.8100 <br>
+            E-mail: ryanrroberts73@gmail.com <br>
```

Comment:
you can create a link here by adding <a href="mailto:ryanrroberts73@gmail.com"> ryanrroberts73@gmail.com </a>